src/components/app/types.ts

```diff
@@ -438,6 +438,7 @@ export enum Nodes {
     Event = 'Event',
     Namespace = 'Namespace',
     Overview = 'Overview',
+    MonitoringDashboard = 'MonitoringDashboard',
```

Comment:
This is a duplicated type which also exist in fe-common, in first place overview and monitoring dashboard are not nodes and secondly had to add here since at some places this type was being used